addons/project_timesheet_holidays/views/hr_holidays_views.xml

```diff
@@ -12,10 +12,10 @@
                         Generate timesheets when validating time off requests of this type
                     </div>
                     <group>
-                        <field name="timesheet_project_id" context="{'active_test': False}"/>
+                        <field name="timesheet_project_id" context="{'active_test': False}" invisible="not company_id"/>
                         <field name="company_id" invisible="1"/>
                         <field name="timesheet_task_id" context="{'active_test': False, 'default_project_id': timesheet_project_id}" invisible="not timesheet_project_id" required="timesheet_project_id"/>
-                        <field name="timesheet_generate" invisible="1"/>
+                        <field name="timesheet_generate" context="{'active_test': False}" invisible="company_id"/>
```

Comment:
The context here is useless, no?

Reply:
yes, my bad, it was copy pasting and playing with fields effect